BackEnd/src/main/java/com/javaschool/entitymapper/MessageType.java

```diff
@@ -0,0 +1,11 @@
+package com.javaschool.entitymapper;
```

Comment:
Why did you named your package as `entitymapper` if your class is a Request message to the queue?